src/utils.ts

```diff
@@ -1,21 +1,55 @@
 'use strict'
 
-import { multihash, blake } from './deps.ts'
+import { base58btc, blake, colors } from './deps.ts'
+import { type Multibase } from './deps.ts'
 
 // TODO: implement a streaming hashing function for large files
-export function blake32Hash (data: string | Uint8Array): string {
+export function blake32Hash (data: string | Uint8Array): Multibase<'z'> {
   // TODO: for node/electron, switch to: https://github.com/ludios/node-blake2
-  const uint8array = blake.blake2b(data, undefined, 32)
-  // if you need Buffer: https://doc.deno.land/https://deno.land/std@0.141.0/io/mod.ts/~/Buffer
-  // const buf = Buffer.from(uint8array.buffer)
-  return multihash.toB58String(multihash.encode(uint8array, 'blake2b-32', 32))
+  const uint8array = typeof data === 'string' ? new TextEncoder().encode(data) : data
+  const digest = blake.blake2b.blake2b256.digest(uint8array)
+  // While `digest.digest` is only 32 bytes long in this case,
+  // `digest.bytes` is 36 bytes because it includes a multiformat prefix.
+  return base58btc.encode(digest.bytes)
```

Comment:
Alright, now it seems to be returning a consistent prefix. But ... why is it a different prefix from the one we had before...? `21XWnNK` vs `z2Drjgb`

Is it because this is a "CID"? Or is it because something in the prefix itself is different (like the type of hash or the length of the hash?)

Reply:
That's because we we're tacking a blake2b-32 prefix code onto 32-byte hashes, as I tried to explain earlier on Slack. So I couldn't reproduce the same hashes as before using only the new library functions. Then I investigated a bit and if I read correctly, blake2b-32 is meant for 32-bit long hashes, not 32-bytes,  so I started using blake2b-256 instead.